ui/src/views/404/__snapshots__/index.spec.jsx.snap

```diff
@@ -2,7 +2,25 @@
 
 exports[`404 generates the expected markup 1`] = `
 Object {
-  "css": "<style class=\\"_styletron_hydrate_\\"></style>",
-  "html": "<div><svg><text>test svg</text></svg><h1>Sorry - we haven’t been able to serve the page you asked for.</h1><p>You may have followed a broken or outdated link, or there may be an error on our site.</p><p>Please follow one of the links below to continue exploring.</p></div>",
+  "css": "<style class=\\"_styletron_hydrate_\\">.a{display:none}</style>",
+  "html": "<div><div><a href=\\"https://www.theguardian.com/\\"><svg><text>test svg</text></svg><span>the Guardian</span></a><svg><text>test svg</text></svg><div><h1>Sorry — the page you have requested does not exist</h1><p>You may have followed an outdated link, or have mistyped a URL. If you believe this to be an error, please<a href=\\"https://www.theguardian.com/info/tech-feedback\\">report it</a>.</p><a href=\\"https://www.theguardian.com/\\">Go to the Guardian home page<svg><text>test svg</text></svg></a></div></div><script>
```

Comment:
Our style is to use an en dash (–) in “Sorry – the”, not an em dash (—), [Style guide](https://www.theguardian.com/guardian-observer-style-guide-d) says if doesn’t exactly follow its own advice...

Plus, is there a space missing between “please” and “report it”?

Reply:
:christmas_tree: toptier

Reply:
@paperboyo I added the space back in my last commit - good spot!

@sndrs Should I update said dash?

Reply:
if it's house style I guess so?